init.yml

```diff
@@ -0,0 +1,164 @@
+##############################################################
+#### ANET Initialization Configuration
+####
+#### This is the configuration file for the ANET db initialization.
+####
+#### For Production use, replace all areas with ${NAME} with
+#### the correct values for your installation.
+####
+#### For Development use, create a file called `localSettings.gradle`
+#### And set each environment variable there
+####  ie: `run.environment('ANET_SMTP_SERVER', 'my-mail-server.example.com`)
+#### Do not make any changes to this file unless they apply to all
+#### developers!
```

Comment:
I don't think these comments are applicable to this log file.

I suggest to replace this with the reason why this log configuration file is different.